src/components/WorkflowEngine/__mocks__/_engineMock.ts

```diff
@@ -0,0 +1,133 @@
+import { EngineInterface, EngineProps } from '../engine'
+import type {
+  WorkflowResponse,
+  OutcomeStepKeys,
+  GetWorkflowFunc,
+  WorkflowStepConfig,
+} from '../utils/WorkflowTypes'
+
+export class MockEngine implements EngineInterface {
+  public engineProps: EngineProps
+
+  constructor(engineProps: EngineProps) {
+    this.engineProps = engineProps
+  }
+
+  getOutcomeStep = (workflow: WorkflowResponse): OutcomeStepKeys => {
+    return !workflow.has_remaining_interactive_tasks
+      ? 'complete'
+      : workflow.outcome
+      ? 'pass'
+      : 'reject'
+  }
+
+  getWorkflow: GetWorkflowFunc = async (): Promise<WorkflowResponse> => {
+    return {
+      id: 'ec9013ea',
+      applicant_id: 'd8034341-5ca2-4f90-a1c6-ae92c9519a21',
+      config: { name: 'timeout', value: 1209600 },
+      //created_at:"2022-03-16T12:08:53.158824",
+      finished: false,
+      // state:"in_progress",
+      task_def_id: 'upload_document_photo',
+      task_id: '2a11059f-b2dd-4374-9e72-58bb2cb410b8',
+      task_type: 'INTERACTIVE',
+      // updated_at:"2022-03-16T12:09:11.972281",
+      // version_id:5,
+      // workflow_id:"5600f9cc-ea4d-4d29-9d23-355005dc7946"
```

Comment:
Are you planning to uncomment these values? If not, can we remove? Same for all other comments in this file

Reply:
removed